packages/backend/src/models/ProjectModel/ProjectModel.ts

```diff
@@ -426,9 +423,8 @@ export class ProjectModel {
 
     async tryWithProjectLock(
```

Comment:
project lock no longer manages jobs at all.

every project id has a lock in postgres

if the function call gets the lock it runs `onLockAcquired` if not then it runs `onLockFailed`

then releases the lock (automatically on transaction end)

Reply:
Nice, we should probably rename the method now then

Reply:
Renamed `tryAcquireProjectLock`